### PR TITLE
59.implement follower map

### PR DESCRIPTION
**Pull request by EPostill**
refer to commits to see what was added

also i added a bias to the location selector

### CODE REVIEW

File: app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java [hunk 1]
```diff
@@ -84,14 +86,22 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         openLocationAutofill.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-
```
Comment (jacobwong98) [inline app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java]: nice

File: app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java [hunk 2]
```diff
@@ -84,14 +86,22 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
...
                 // Set the fields to specify which types of place data to
                 // return after the user has made a selection.
                 List<Place.Field> fields = Arrays.asList(Place.Field.ID, Place.Field.NAME, Place.Field.LAT_LNG);
 
+                //get location for bias area
+                Location currentLoc = LocationController.getUserLocation(getContext());
+                double currentLat = currentLoc.getLatitude();
+                double currentLng = currentLoc.getLongitude();
+                double biasOffset = 0.1;
```
Comment (jacobwong98) [inline app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java]: why 0.1?

File: app/src/main/java/com/example/vybe/MapViewActivity.java [hunk 3]
```diff
@@ -112,6 +121,38 @@ public void addFollowedVibeLocations() {
...
+                            public void onSuccess(QuerySnapshot queryDoc) {
+                                for (QueryDocumentSnapshot document: queryDoc){
+                                    if ((document.getDouble("latitude") != 0) && (document.getDouble("latitude") != 0)) {
+                                        double latitude = document.getDouble("latitude");
+                                        double longitude = document.getDouble("longitude");
+                                        String vibeName = (String) document.getData().get("vibe");
+                                        Vibe vibe = VibeFactory.getVibe(vibeName);
+
+                                        mapFragment.addMarker(new LatLng(latitude, longitude), vibe.getEmoticon());
+
```
Comment (jacobwong98) [inline app/src/main/java/com/example/vybe/MapViewActivity.java]: Need to stub this out when we have the social list done

File: app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java (hunk 1 above)
Comment (EPostill) [reply app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java]: thanks

File: app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java (hunk 2 above)
Comment (EPostill) [reply app/src/main/java/com/example/vybe/AddEdit/LocationSelectionDialog.java]: because i dont like 0.099

**Pull request merged**